src/conv-ldb/generate.C

```diff
@@ -122,7 +122,7 @@ n -= (V-1);
 }
 #endif
 
-static void AddEdges(VerticesListType *graph, EdgeListType *EdgeList, int V, int n)
+static void AddEdges(VerticesListType *graph, EdgeListType *EdgeList, int v, int n)
```

Comment:
Since we're bulk renaming stuff here anyway, it might be worth changing their names to things that make more semantic sense, like changing this from `V` to `numvertices` or something of that ilk.